boot/zephyr/Kconfig

```diff
@@ -194,10 +194,21 @@ config BOOT_VALIDATE_SLOT0_ONCE
 	  low end devices with as a compromise lowering the security level.
 	  If unsure, leave at the default value.
 
+config BOOT_PREFER_SWAP_MOVE
```

Comment:
I would name this option with prefix MCUBOOT instead of BOOT - reason is that this will be used from boards defconfig files in zephyr, this would be right visible that it is for MCUBOOT.

Reply:
@d3zd3z  One nit ^^

Reply:
Hmm, it makes it different than the rest of the options, but I can certainly do it that way if that makes it clearer.

Reply:
Thinking more about this. I don't think it makes sense to add an MCUBOOT prefix to this, unless we change all of the rest of the options as well. This is the Zephyr config file, and any of these will be showing up in board defconfigs.

Reply:
okay, I won't insist